### PR TITLE
feature: Add time to first/last approve/review

### PR DESCRIPTION
**Pull request by rtfpessoa**

### CODE REVIEW

**Comment by rtfpessoa** [comment]
It just caught a broken link, amazing that the linters can catch this. cc\ @prcr

**Review by mrfyda — APPROVED** [review]

File: docs/metrics/lead-time-reviews.md [hunk 1]
```diff
@@ -32,3 +32,17 @@ This metric is calculated as the time between the pull request being created and
...
+
+This metric is calculated as the time between the pull request being created and the first and last approves being submitted.
+
+### Time to first/last review
+
+The time it takes to have reviews. Once a pull request is opened, how long does it take to have a review and how long does it take to the last review?
+
+This metric is calculated as the time between the pull request being created and the first and last reviews being submitted.
+
+Reviews include approves, changes requested and code comments, does not include pull request comments.
```
Comment (mrfyda) [inline docs/metrics/lead-time-reviews.md:48]:
```suggestion
Reviews include approves, changes requests and inline pull request comments, does not include conversation pull request comments.
```

File: docs/metrics/lead-time-reviews.md [hunk 2]
```diff
@@ -32,3 +32,17 @@ This metric is calculated as the time between the pull request being created and
...
+
+This metric is calculated as the time between the pull request being created and the first and last approves being submitted.
+
+### Time to first/last review
+
+The time it takes to have reviews. Once a pull request is opened, how long does it take to have a review and how long does it take to the last review?
+
+This metric is calculated as the time between the pull request being created and the first and last reviews being submitted.
+
+Reviews include approves, changes requests and inline pull request comments, does not include conversation pull request comments.
```
Comment (prcr) [inline docs/metrics/lead-time-reviews.md:48]:
```suggestion
Reviews include approves, change requests, and inline pull request comments, but don't include conversation pull request comments.
```

**Review by prcr — APPROVED** [review]
This is a great iteration. :shipit: :+1: 

I'm starting to entertain the idea of having a diagram/timeline to illustrate all the time periods covered by the "Lead time and reviews" group of metrics, it could be a nice addition next.

**Pull request merged**